velox/common/memory/ByteStream.h

```diff
@@ -247,6 +247,11 @@ class ByteOutputStream {
   }
 
   void startWrite(int32_t initialSize) {
+    ranges_.clear();
```

Comment:
How about put this into reset()? Or do we allow to call startWrite more than once? Thanks!